io.github.eclab.edisyn.yml

```diff
@@ -0,0 +1,44 @@
+
+id: io.github.eclab.edisyn
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.openjdk
+command: run.sh
+modules:
+    -   name: openjdk
+        buildsystem: simple
+        build-commands:
+            -   /usr/lib/sdk/openjdk/install.sh
+
+    -   name: edisyn
+        buildsystem: simple
+        build-commands:
+            -   install -Dm755 run.sh $FLATPAK_DEST/bin/run.sh
+            -   install -Dm755 edisyn.jar $FLATPAK_DEST/bin/edisyn.jar
+            -   install -D io.github.eclab.edisyn.desktop $FLATPAK_DEST/share/applications/io.github.eclab.edisyn.desktop
+            -   install -D io.github.eclab.edisyn.svg $FLATPAK_DEST/share/icons/hicolor/scalable/apps/io.github.eclab.edisyn.svg
+            -   install -D io.github.eclab.edisyn.metainfo.xml $FLATPAK_DEST/share/metainfo/io.github.eclab.edisyn.metainfo.xml
+        sources:
+            -   type: file
+                url: https://cs.gmu.edu/~eclab/projects/edisyn/edisyn.jar
+                sha256: f6f44f4e0de39ace25cfc46891f378d7d715a1ff1387632a39369bb20ade71b6
+                dest-filename: edisyn.jar
+            -   type: file
+                path: run.sh
+            -   type: file
+                path: io.github.eclab.edisyn.desktop
+            -   type: file
+                path: io.github.eclab.edisyn.svg
+            -   type: file
+                path: io.github.eclab.edisyn.metainfo.xml
+
+                
+finish-args:
+    - --env=PATH=/app/jre/bin:/usr/bin
+    - --socket=x11
+    - --share=ipc
+    - --device=all
```

Comment:
why do you need device=all?

Reply:
because the app needs access to ALSA or Jack. But --socket=pulseaudio may also work as I have learned

Reply:
this will not provide JACK. You need to the pipewire socket for that.
As for ALSA, the pulseaudio permission provides it.

Reply:
There is no dedicated pipewire socket. Pipewire  provides Pulseaudio. So we'll go with that. I updated the sandbox permissions accordingly. Should be fine now.

Reply:
> There is no dedicated pipewire socket.

False.